kong/plugins/request-size-limiting/handler.lua

```diff
@@ -2,16 +2,16 @@
 
 local BasePlugin = require "kong.plugins.base_plugin"
 local responses = require "kong.tools.responses"
-local stringy = require "stringy"
+local strip = require("pl.stringx").strip
 
 local RequestSizeLimitingHandler = BasePlugin:extend()
 
 RequestSizeLimitingHandler.PRIORITY = 950
 
 local function check_size(length, allowed_size, headers)
-  local allowed_bytes_size = allowed_size * 100000
+  local allowed_bytes_size = allowed_size * 1000000
```

Comment:
I find it much more convenient and requiring less cognitive load (as well as much less error prone) to declare constants representing size (including MBs) like so:
```lua
local CONST_NAME = X * 10^6 -- here X being 1
```
Then use `CONST_NAME` in the module's function.